frontend/src/components/TriggeredPrompt.tsx

```diff
@@ -56,113 +29,101 @@ const TriggeredPrompt: FunctionComponent<TriggeredPromptProps> = ({
   promptDelay,
   secondPromptFinishedHandler,
 }): ReactElement => {
-  let [canPlayPrompt1, setCanPlayPrompt1] = useState(false);
-  let [canPlayPrompt2, setCanPlayPrompt2] = useState(false);
-  let [prompt1Duration, setPrompt1Duration] = useState(0);
-  let [prompt2Duration, setPrompt2Duration] = useState(0);
-
-  const updatePrompt1 = (canPlay: boolean, duration: number) => {
-    setPrompt1Duration(duration);
-    setCanPlayPrompt1(canPlay);
-  };
-
-  const updatePrompt2 = (canPlay: boolean, duration: number) => {
-    setPrompt2Duration(duration);
-    setCanPlayPrompt2(canPlay);
-  };
-
-  const prompt1 = new Audio(prompt1Url);
-  const prompt2 = new Audio(prompt2Url);
-
-  prompt2.onended = () => {
-    secondPromptFinishedHandler?.();
-  };
-
-  prompt1.preload = "metadata";
-  prompt2.preload = "metadata";
-
-  useEffect(() => {
-    //loop prompt if this is an assessment
-    getDuration(prompt1, 0, 50, updatePrompt1);
-
-    if (isAssessment && canPlayPrompt1) {
-      playPrompt1();
-      const interval = setInterval(
-        playPrompt1,
-        prompt1Duration * 1000 + 8000
-      );
-      return () => {
-        stopAudio(prompt1);
-        stopAudio(prompt2);
-        clearInterval(interval);
-      };
+  const [audio1, setAudio1] = useState(new Audio(prompt1Url));
```

Comment:
in the code I'm getting that the variables promptDelay and currentWord are never used  by TriggeredPrompt -- might as well just remove it then if that's still the case